src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)];
     }
 }
```

Comment:
The class `ColorSupplier` should have an access modifier. It is good practice to define it explicitly to adhere to encapsulation principles.